File: test1q1.java
/**
 *
 * @author adanasir
 */
package test1q1;

import java.io.FileNotFoundException;
import java.io.IOException;
import java.io.UnsupportedEncodingException;
import java.io.PrintWriter;


public class test1q1 {
   
        public static void main(String[] args) throws FileNotFoundException, UnsupportedEncodingException, IOException {
              
        String p = "Name: Siti Aminah Shuhada Mat Nasir. Matrix No.: 2014433638. Gender: Female. Age: 32 years old. E-mail: adanasir@gmail.com. Mobile Phone: 0199171336";
        byte[] buffer = p.getBytes();
        
        StringBuilder binary = new StringBuilder();
        for (byte b : buffer){
            int val = b;
            for (int a = 0; a < 8; a++)
            {
                binary.append((val & 128) == 0 ? 0 : 1);
                val <<= 1;
            }
            binary.append(' ');      
        }
              
        //System.out.println("'" + p + "'to binary" + binary);
        
        try (
        PrintWriter writer = new PrintWriter("C:/Users/Administrator/Desktop/adanasirjava.bin", "UTF-8")) {
             //writer.println("'" + p + "'" + " to binary = " + binary);
             writer.println(binary);
        }
    }
}

File: python/test1q1.py
#
#SITI AMINAH SHUHADA MAT NASIR
#2014433638
#adanasir@gmail.com
#github.com/adanasir

def main():

    buffersize = 50000

    infile=open("adanasirpy.txt", "rb")
    outfile=open("adanasirpy.bin", "wb")

    buffer = infile.read(buffersize)

    while len(buffer):
        outfile.write(buffer)
        print(".", end = " ")
        buffer = infile.read(buffersize)
    print()
    print("Done")
    
infile.close()

#file = open("", "w")
#file.write("Name: Siti Aminah Shuhada Mat Nasir\n")
#file.write("Matrix No.: 2014433638\n")
#file.write("Gender: Female\n")
#file.write("Age: 32 years old\n")
#file.write("E-mails: adanasir@gmail.com\n")
#file.write("Mobile Phone: 0199171336\n")
#file.close()